packages/hash/external-services/docker-compose.test.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+
+volumes:
+  hash-postgres-test-data:
+
+services:
+  postgres:
+    environment:
+      HASH_KRATOS_PG_DATABASE: "${HASH_KRATOS_PG_TEST_DATABASE}"
+      HASH_GRAPH_PG_DATABASE: "${HASH_GRAPH_PG_TEST_DATABASE}"
+    volumes:
+      - hash-postgres-test-data:/var/lib/postgresql/data
```

Comment:
Do we need to mount a volume here? I think it would work fine without. Or do we wish to persist the postgres data between runs?

Reply:
Fair point, I've removed it in https://github.com/hashintel/hash/pull/1077/commits/8e4a2243aaf220407c0afda480a05af52bdf4d4e